src/Solnet.Rpc/Utilities/AddressExtensions.cs

```diff
@@ -18,23 +20,23 @@ public static class AddressExtensions
         /// <exception cref="Exception">Throws exception when the resulting address doesn't fall off the Ed25519 curve.</exception>
         public static byte[] CreateProgramAddress(IList<byte[]> seeds, byte[] programId)
         {
-            var buffer = new List<byte>();
+            var buffer = new MemoryStream();
```

Comment:
Initialize the MemoryStream with (32*seed + "ProgramDerivedAddress".Length + programId.Length) as initial capacity.

Reply:
This is done